dist/common/scripts/scylla_util.py

```diff
@@ -99,6 +100,29 @@ def scyllabindir():
 def sysconfdir():
     return str(sysconfdir_p())
 
+def get_required_aio_slots():
+    return cpu_count() * 11026 + 65536
+
+def configure_aio_slots():
+    aio_conf_file = "/etc/sysctl.d/aio-max-nr.conf"
+    aio_max_nr_value = "fs.aio-max-nr = " + str(get_required_aio_slots())
+    mode = 'w+'
+    if os.path.exists(aio_conf_file):
+        with open(aio_conf_file, 'r') as file:
+            for line in file:
+                aio_conf_value = re.findall(r"[=]?\d*\.\d+|\d+", line)
+                aio_value = int((', '.join(map(str, aio_conf_value))))
+                if aio_value > get_required_aio_slots():
+                    with open('/proc/sys/fs/aio-max-nr', mode) as f:
+                        f.write(str(aio_value))
+    else:
+        with open(aio_conf_file, mode) as f:
+            f.write(aio_max_nr_value + '\n')
+            with open('/proc/sys/fs/aio-max-nr') as f:
+                aio_max_nr = int(f.read())
+                with open('/proc/sys/fs/aio-max-nr', mode) as f:
+                    f.write(str(get_required_aio_slots()))
```

Comment:
i think indent is need here or the name of open file descriptor  for each context manager should be changed.
Because the latest context manger will rewrite first one and first file will not be closed.

Reply:
Fixed